### ISSUE TITLE
[PR] Keyboard manoeuvre : open items

### ISSUE THREAD

**Issue by dstala** [nocodb/nocodb#4407]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

Open items copied from here
https://github.com/nocodb/nocodb/pull/4222#issuecomment-1301940952

### Shortcut for often repeated actions in database
- [x] Add new table ( alt + t )
- [x] Add new row ( alt + r )
- [x] Add new column ( alt + c )
- [x] Toggle full screen mode ( alt + f )
- [x] Team & Settings ( alt + , )
- [x] Invite Team ( alt + i )
- [x] Expanded form save (cmd+enter or ctrl+enter)
- [x] Cmd Left-arrow Move to leftmost cell in selected row : navigates route (browser behaviour?)
- [x] Cmd Right-arrow Move to rightmost cell in selected row : navigates route (browser behaviour?)
- [ ] Shift Enter Insert a record below selected cell **(on hold - wait for PR4504)**
- [x] Cmd A to select all cell in a text when in edit mode (select-all) : double click on text cell (or other editable cell), press `Cmd A`

### Shortcut specific to datatype
- [x] `date` : press `space` for expand row. press `enter` : date picker pops up. picker shouldn't as focus is expected to be on expanded row
- [x] `select` column: click to select cell on which a value is already configured, press `enter`; dropdown didn't appear
- [x] change warning message as: `Computed field: contents are read-only. Use column edit menu to reconfigure`
- [x] space bar opens expanded row drawer. ensure focus shifts to expanded row drawer until its closed (using esc or manually). input focus can be first cell; if its easy (preferably) on the cell from where space bar was pressed
- [x] `attachment`: enter key opens attachment modal. ensure focus is on attachment modal once its open. pressing `enter` key again should emulate `attach file` button click
- [ ] ~~`date` column: enter opens picker. see if we can focus on picker so that if i press enter again, either `now` or current date time is selected. and if i press tab, it moves around major keys of picker modal~~
- [x] Percent column: unable to edit on enter key press. double click also had some issue to edit cell contents
- [x] `Lookup` : double click, warning is not displayed anymore
- [x] System fields `CreatedAt`, `UpdatedAt`: mark them as read only for all roles
- [x] Date: click on date cell, `Cmd + ;` should populate with current date &/ time (= emulate `now` button)
- [x] JSON: `Enter` key toggles to edit mode. In edit mode, focus should be on input text box. We should be able to start typing to add/remove/append text. Currently we need an additional mouse click before we start editing JSON cell contents. 
- [x] JSON: In edit mode - add support for `shift enter` to insert new line. its mapped to `cmd enter` currently

### Project Details

feat/keyboard-manoeuvre

**Comment by o1lab** [comment]
@mustafapc19 : can u pls add tests for this as well.
